views/pages/login.ejs

```diff
@@ -27,6 +41,64 @@
                 </select>
               </div>
             </div>
+
+            <div class="field">
+              <label class="label">
+                Scopes
+                <span class="has-text-info tooltip is-tooltip-multiline"
+                  data-tooltip="Vous pouvez régler les scopes voulues pour la procédure">
+                  (info)
+                </span>
+              </label>
+              <div class="block">
+                <div class="select">
+                  <div class="control">
+                    <select id="scope_menu" value="profile">
+                      <option value="profile">Défaut : Profile</option>
+                      <option value="identity">Identité Pivot</option>
+                      <option value="birth">Naissance</option>
+                      <option value="basics">Basiques</option>
```

Comment:
this basics option should be named « Tous » because it actually return all the data available via FranceConnect.